### ISSUE TITLE
[grafana] release version 6.25.0 not present in helm repo

### ISSUE THREAD

**Issue by chlumper** [grafana/helm-charts#1171]
The version `6.25.0` of the grafana helm chart is not available in the helm chart repository (https://grafana.github.io/helm-charts) but listed on the release page. There was an "already exists" issue on the first release pipeline run which I think @Xtigyro tried to resolve by re-queueing but on the 2nd run no release was created since releaser thought there is nothing to do.

* pipeline release run 1:
https://github.com/grafana/helm-charts/runs/5801614341?check_suite_focus=true
failed with
`422 Validation Failed [{Resource:Release Field:tag_name Code:already_exists Message:}]`

* pipeline release run 2:
https://github.com/grafana/helm-charts/runs/5801642178?check_suite_focus=true
output from chart-releaser
  ```
  Looking up latest tag...
  Discovering changed charts since 'grafana-6.25.0'...
  Nothing to do. No chart changes detected.
  ```

* release/tag:
https://github.com/grafana/helm-charts/releases/tag/grafana-6.25.0

* repo index (still referencing `6.24.1`):
https://github.com/grafana/helm-charts/blob/795f1b39937aeaeb24b44daa726b815e24e8a510/index.yaml#L1832

**Comment by Xtigyro** [comment]
Hey @chlumper - seems you're right. Looking into how we can still fix this.

**Comment by Xtigyro** [comment]
A possible fix might be deleting the actual `6.25.0` release of the Grafana chart and the tag for it - and re-running the pipeline.

@zanhsieh What do you think?

**Comment by Xtigyro** [comment]
@chlumper Is it acceptable for ya (in the meantime) to use: https://github.com/grafana/helm-charts/releases/tag/grafana-6.25.1?

**Comment by chlumper** [comment]
@Xtigyro Yes using the `6.25.1` is totally fine! Thank you for providing the new version.

**Comment by chlumper** [comment]
If you still want to fix `6.25.0` feel free to re-open the issue.
